test/cookbooks/docker_test/recipes/network.rb

```diff
@@ -225,3 +225,19 @@
   network_name 'network_h1'
   action :disconnect
 end
+
+###########
+# network_i
+###########
+
+# IPv6 enabled network
+docker_network 'network_i1' do
```

Comment:
My intent was to keep the naming consistant with the Docker API https://docs.docker.com/engine/api/v1.30/#operation/NetworkCreate - the docker network CLI option is "--ipv6".